### PR TITLE
StorefrontAccount: Add isVerified property and remove customFields

### PR DESCRIPTION
**Pull request by mehicalen**
## Summary
This PR removed `customFields` property and adds missing `isVerified` property to StorefrontAccount schema.

## Checklist

- [x] Writing style
- [x] API design standards

### CODE REVIEW

File: openapi/components/schemas/StorefrontAccount.yaml [hunk 1]
```diff
@@ -5,6 +5,9 @@ properties:
     readOnly: true
     allOf:
       - $ref: ./ResourceId.yaml
+  isVerified:
+    description: Email verification status.
```
Comment (ashkarpetin) [inline openapi/components/schemas/StorefrontAccount.yaml:9]:
```suggestion
    description: Email verification status.
    type: boolean
```

Comment (mehicalen) [reply openapi/components/schemas/StorefrontAccount.yaml:10]: my bad, updated :) Thank you!

**Review by ashkarpetin — DISMISSED** [review]

**Review by mikerebilly — DISMISSED** [review]

**Review by nanobash — DISMISSED** [review]

File: openapi/components/schemas/StorefrontAccount.yaml [hunk 2]
```diff
@@ -5,6 +5,10 @@ properties:
     readOnly: true
     allOf:
       - $ref: ./ResourceId.yaml
+  isVerified:
+    type: boolean
+    description: Email verification status.
```
[the remark this answers is not in the document]
Comment (mehicalen) [reply openapi/components/schemas/StorefrontAccount.yaml]: Thank you, updated!

**Review by noon-dawg — DISMISSED** [review]

**Comment by mehicalen** [comment]
@php please review

**Review by ashkarpetin — DISMISSED** [review]

**Pull request closed without merging**